lib/cmd/fh3/app/create.js

```diff
@@ -56,6 +62,14 @@ module.exports = {
 
       setAutoDeploy(params);
       return cb(null, params);
+    } if (argv.empty){
+      params.template = {
+        autoDeployOnCreate : "demos-dev",
```

Comment:
We cannot get this into the master. It would be better to make this solution more generic if we want to merge this back.